### ISSUE TITLE
Renewal doesn't work

### ISSUE THREAD

**Issue by argusua** [jetstack/kube-lego#51]
Version of kube-lego:

```
2016-10-24T13:22:49.775587542Z time="2016-10-24T13:22:49Z" level=info msg="kube-lego 0.1.2-a823e819 starting" context=kubelego 
2016-10-24T13:22:49.804969421Z time="2016-10-24T13:22:49Z" level=info msg="connected to kubernetes api v1.4.3" context=kubelego 

```

Logs for one domain:

```
2016-10-24T13:23:22.062415223Z time="2016-10-24T13:23:22Z" level=info msg="cert expires soon so renew" context="ingress_tls" expire_time=2016-11-14 16:27:00 +0000 UTC name=kubedash namespace=kube-system 
2016-10-24T13:23:22.062443256Z time="2016-10-24T13:23:22Z" level=info msg="requesting certificate for kubedash.test.gelato.tech" context="ingress_tls" name=kubedash namespace=kube-system 
2016-10-24T13:23:25.336489023Z time="2016-10-24T13:23:25Z" level=info msg="authorization successful" context=acme domain=kubedash.test.gelato.tech 
2016-10-24T13:23:25.336529610Z time="2016-10-24T13:23:25Z" level=warning msg="authorization failed for some domains" context=acme failed_domains=[ kubedash.test.gelato.tech] 

```

And one more error from logs:
`2016-10-24T13:36:36.966341172Z time="2016-10-24T13:36:36Z" level=error msg="Error while process certificate requests: error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required, error getting certificate: 400 urn:acme:error:malformed: Error creating new cert :: at least one DNS name is required" context=kubelego`

**Comment by simonswine** [comment]
@argusua can you retry this with the referenced fix?

It's available through `jetstack/kube-lego:canary`

**Comment by argusua** [comment]
Looks like works now! Thank You.
```
2016-11-03T12:18:06.474069142Z time="2016-11-03T12:18:06Z" level=info msg="kube-lego 0.1.2-00d38b4b starting" context=kubelego
....
2016-11-03T12:18:06.539700306Z time="2016-11-03T12:18:06Z" level=info msg="cert expires soon so renew" context="ingress_tls" expire_time=2016-11-14 16:27:00 +0000 UTC name=kubedash namespace=kube-system 
2016-11-03T12:18:06.539773681Z time="2016-11-03T12:18:06Z" level=info msg="requesting certificate for kubedash.test.gelato.tech" context="ingress_tls" name=kubedash namespace=kube-system 
2016-11-03T12:18:40.051638499Z time="2016-11-03T12:18:40Z" level=info msg="authorization successful" context=acme domain=kubedash.test.gelato.tech 
2016-11-03T12:18:40.914159609Z time="2016-11-03T12:18:40Z" level=info msg="successfully got certificate: domains=[kubedash.test.gelato.tech] url=https://acme-v01.api.letsencrypt.org/acme/cert/03ad38c2d4effdd28ec22a99c5977c8b5b3c" context=acme
```

**Comment by simonswine** [comment]
Ok I released this as part of 0.1.3
